### PR TITLE
Fix intercom_flutter pod version

### PR DESCRIPTION
**Pull request by gabdsg**

### CODE REVIEW

**Review by Zazo032 — APPROVED** [review]

File: intercom_flutter/ios/intercom_flutter.podspec [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'intercom_flutter'
-  s.version          = '4.0.0'
+  s.version          = '5.0.0'
```
Comment (deepak786) [inline intercom_flutter/ios/intercom_flutter.podspec:6]: Why it is required to update?

Comment (gabdsg) [reply intercom_flutter/ios/intercom_flutter.podspec:6]: Fixes an issue when building in CodeMagic, not sure why

Comment (deepak786) [reply intercom_flutter/ios/intercom_flutter.podspec:6]: Does that mean this version should be updated every time as the new version of the package is released?

Comment (deepak786) [reply intercom_flutter/ios/intercom_flutter.podspec:6]: @Zazo032 @v3rm0n should we merge this change? In fact, I didn't understand why it is required?

Comment (Zazo032) [reply intercom_flutter/ios/intercom_flutter.podspec:6]: It's not required, for example, you can check the `battery` [plugin](https://github.com/flutter/plugins/blob/master/packages/battery/battery/ios/battery.podspec), pub version is 2.0.3 and podspec file still has 0.0.1

Comment (gabdsg) [reply intercom_flutter/ios/intercom_flutter.podspec:6]: I did more testing and indeed this was not related to the error... but it's nice to have it with the same version...

Comment (deepak786) [reply intercom_flutter/ios/intercom_flutter.podspec:6]: @Zazo032 what do think then? Should we update the version?

Comment (Zazo032) [reply intercom_flutter/ios/intercom_flutter.podspec:6]: We can merge the change, but it won't have any functional change

Comment (deepak786) [reply intercom_flutter/ios/intercom_flutter.podspec:6]: 👍

**Pull request merged**